dev-haskell/plugins/plugins-1.6.2.1.ebuild

```
@@ -0,0 +1,38 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Dynamic linking for Haskell and C objects"
+HOMEPAGE="https://github.com/stepcut/plugins"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+SRC_URI="${SRC_URI}
+	https://github.com/stepcut/plugins/pull/21.patch"
```

Make this a less generic filename, please.

```suggestion
SRC_URI+="
	https://github.com/stepcut/plugins/pull/21.patch
		-> ${P}-ghc-9_8.patch
"